integreat_cms/cms/forms/events/event_filter_form.py

```diff
@@ -148,6 +158,22 @@ def apply(
         elif recurrence.NOT_RECURRING in self.cleaned_data["recurring"]:
             # Only non-recurring events
             events = events.filter(recurrence_rule__isnull=True)
+
+        if (
+            len(self.cleaned_data["imported_event"]) == len(calendar.CHOICES)
+            or not self.cleaned_data["imported_event"]
+        ):
+            pass
```

Comment:
Could you add a comment about this? Maybe not necessarily right above this, but since the pattern is repeated multiple times in that function, maybe above the first occurrence? With the context "check if either all options or none were selected from this group" it makes perfect sense, but IMO not super intuitive.